.circleci/config.yml

```diff
@@ -1,318 +1,43 @@
-                paths:
-                    - ~/nilearn_data/brainomics_localizer # ~ 66 MB
-
-            - save_cache:
-                key: v1-development_fmri-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/development_fmri     # ~373 MB
-
-            - save_cache:
-                key: v1-ds000030-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/ds000030             # ~832 MB
-
-            - save_cache:
-                key: v1-fiac_nilearn.glm-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/fiac_nilearn.glm     # ~ 82 MB
-
-            - save_cache:
-                key: v1-fMRI-language-localizer-demo-dataset-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/fMRI-language-localizer-demo-dataset # ~ 750 MB
-
-            - save_cache:
-                key: v1-fsl-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/fsl                  # ~ 30 MB
-
-            - save_cache:
-                key: v1-haxby2001-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/haxby2001            # ~302 MB
-
-            - save_cache:
-                key: v1-icbm152_2009-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/icbm152_2009         # ~ 63 MB
-
-            - save_cache:
-                key: v1-jimura_poldrack_2012_zmaps-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/jimura_poldrack_2012_zmaps # ~ 105 MB
-
-            - save_cache:
-                key: v1-localizer_first_level-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/localizer_first_level # ~ 35 MB
-
-            - save_cache:
-                key: v1-miyawaki2008-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/miyawaki2008          # ~181 MB
-
-            - save_cache:
-                key: v1-nki_enhanced_surface-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/nki_enhanced_surface  # ~ 85 MB
-
-            - save_cache:
-                key: v1-oasis1-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/oasis1                # ~913 MB
-
-            - save_cache:
-                key: v1-smith_2009-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/smith_2009            # ~171 MB
-
-            - save_cache:
-                key: v1-spm_auditory-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/spm_auditory          # ~230 MB
-
-            - save_cache:
-                key: v1-spm_multimodal_fmri-{{ checksum "week_num" }}
-                paths:
-                    - ~/nilearn_data/spm_multimodal_fmri   # ~227 MB
```

Comment:
IIUC, all the pre-caching done here is no longer done with Github actions, right ?

Reply:
That's right, I haven't adapted the pre-caching of the data yet but it is on the todo list. I'm just trying to figure out the best to do it for the setup of the Actions workflow.